thoth/storages/graph/postgres.py

```diff
@@ -1651,7 +1672,7 @@ def _construct_si_unanalyzed_python_package_versions_query(
             query = query.filter(PythonPackageIndex.url == index_url)
 
         # We find all rows that are same in PythonPackageVersion and SIAggregated table.
-        conditions = [PythonPackageVersion.id == SIAggregated.python_package_version_id]
+        conditions = [PythonPackageVersion.entity_id == SIAggregated.python_package_version_entity_id]
```

Comment:
@pacospace could you please explain why we switched back to comparing entity_id?

Reply:
Because for each `entity_id` exist `number of solvers x python_package_version_id` therefore when we query for PPV and SI we always receive still the same python_package_version because it exist for different solvers, that is why the number fo SI Unanalyzed was not going down, unless we look for PPVE record for the PPV, because that is one.

Reply:
Makes sense, thanks!

Reply:
If I'm not clear I can draw something, let me know :)